packages/odyssey-storybook/src/components/odyssey-mui/Typography/Typography.stories.tsx

```diff
@@ -26,9 +26,24 @@ import {
   TypographyProps,
   typographyColorValues,
   typographyVariantMapping,
+  TypographyVariantValue,
```

Comment:
```suggestion
  type TypographyVariantValue,
```

We should do this to other types we're importing as well. Helps the compiler.